src/structure/model.jl

```diff
@@ -663,463 +956,658 @@ Base.eltype(::Type{Residue}) = AbstractAtom
-    println(io, "Number of atoms             -  ", countatoms(chain, stdatomselector))
-    println(io, "Number of hydrogens         -  ", countatoms(chain, stdatomselector, hydrogenselector))
-    println(io, "Number of disordered atoms  -  ", countatoms(chain, stdatomselector, disorderselector))
+function Base.show(io::IO, ch::Chain)
+    println(io, "Chain ID                    -  ", chainid(ch))
+    println(io, "Number of residues          -  ", countresidues(ch, stdresselector))
+    println(io, "Number of point mutations   -  ", countresidues(ch, stdresselector, disorderselector))
+    println(io, "Number of other molecules   -  ", countresidues(ch, hetresselector) - countresidues(ch, hetresselector, waterselector))
+    println(io, "Number of water molecules   -  ", countresidues(ch, hetresselector, waterselector))
+    println(io, "Number of atoms             -  ", countatoms(ch, stdatomselector))
+    println(io, "Number of hydrogens         -  ", countatoms(ch, stdatomselector, hydrogenselector))
+    println(io, "Number of disordered atoms  -  ", countatoms(ch, stdatomselector, disorderselector))
 end
 
 function Base.show(io::IO, res::Residue)
-    println(io, "Residue ID                  -  ", resid(res; full=true))
+    println(io, "Residue ID                  -  ", resid(res, full=true))
     println(io, "Residue name                -  ", resname(res))
     println(io, "Number of atoms             -  ", countatoms(res))
     println(io, "Number of hydrogens         -  ", countatoms(res, hydrogenselector))
     println(io, "Number of disordered atoms  -  ", countatoms(res, disorderselector))
 end
 
-function Base.show(io::IO, disordered_res::DisorderedResidue)
-    println(io, "Residue ID                  -  ", resid(disordered_res; full=true))
-    for res_name in resnames(disordered_res)
+function Base.show(io::IO, dis_res::DisorderedResidue)
+    println(io, "Residue ID                  -  ", resid(dis_res, full=true))
+    for res_name in resnames(dis_res)
         println(io, "Residue name                -  ", res_name)
-        println(io, "Number of atoms             -  ", countatoms(disorderedres(disordered_res, res_name)))
-        println(io, "Number of hydrogens         -  ", countatoms(disorderedres(disordered_res, res_name), hydrogenselector))
-        println(io, "Number of disordered atoms  -  ", countatoms(disorderedres(disordered_res, res_name), disorderselector))
+        println(io, "Number of atoms             -  ", countatoms(disorderedres(dis_res, res_name)))
+        println(io, "Number of hydrogens         -  ", countatoms(disorderedres(dis_res, res_name), hydrogenselector))
+        println(io, "Number of disordered atoms  -  ", countatoms(disorderedres(dis_res, res_name), disorderselector))
     end
 end
 
-Base.show(io::IO, atom::Atom) = println(io, pdbline(atom)...)
-Base.showcompact(io::IO, atom::Atom) = print(io, pdbline(atom)...)
+Base.show(io::IO, at::Atom) = println(io, pdbline(at)...)
```

Comment:
`show` should print "an informative text representation of a value" (from the help of `Base.show`) rather than a PDB-formatted text. Informative text would be something like [this](https://github.com/BioJulia/Bio.jl/blob/master/src/align/hts/sam/record.jl#L48-L62).